bin/local-env/install-wordpress.sh

```diff
@@ -90,6 +90,7 @@ echo -e $(status_message "Import default set of media assets...")
 wp media import /var/www/html/wp-content/e2e-assets/example-1.jpg
 wp media import /var/www/html/wp-content/e2e-assets/example-2.jpg
 wp media import /var/www/html/wp-content/e2e-assets/example-3.png
+wp media import /var/www/html/wp-content/e2e-assets/small-video.mov
```

Comment:
```suggestion
wp media import /var/www/html/wp-content/e2e-assets/small-video.mov
wp media import /var/www/html/wp-content/e2e-assets/small-video.mp4
```

Is it worth adding an MP4 here, so we can see that both jpg and mp4 are both working as expected?

Reply:
I think that's probably worth it, it's an easy test. :)

Reply:
Let's add this then.